pkg/maps/lbmap/lbmap.go

```diff
@@ -530,7 +530,7 @@ func serviceKeynValue2FEnBE(svcKey ServiceKey, svcValue ServiceValue) (*loadbala
 	}
 
 	feL3n4Addr := serviceKey2L3n4Addr(svcKey)
-	beLBBackEnd := loadbalancer.NewLBBackEnd(loadbalancer.TCP, beIP, bePort, beWeight)
+	beLBBackEnd := loadbalancer.NewLBBackEnd(loadbalancer.NONE, beIP, bePort, beWeight)
```

Comment:
Not blocker but in reality this should be `loadbalancer.ANY`

Reply:
Agreed but I'm not going to refactor code in this fix ;-)